my_list.rb

```diff
@@ -0,0 +1,17 @@
+require './my_enumerable'
+
```

Comment:
- [ ] Kindly use `require_relative` because using `require_relative` for files in your directory instead of `require` is a ruby convention. "`Require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."

Reply:
Fixed.